src/page/ProductList.page.ts

```diff
@@ -0,0 +1,11 @@
+import { $, ElementFinder, promise } from 'protractor';
+
+export class ProductListPage {
+  private get firstProduct(): ElementFinder {
```

Comment:
use a suffix in order to know what kind of locator it is. for example, Item, Button

Reply:
done